README.md

```diff
@@ -38,6 +38,9 @@ module "openvpn" {
 | ubuntu\_ami\_id | Custom Ubuntu AMI id for the OpenVPN instance. | `string` | `""` | no |
 | volume\_path | Path to mount the data fs. | `string` | `"/openvpn/"` | no |
 | vpc\_id | VPC id where the OpenVPN will be hosted. | `string` | ` ` | yes |
+| backup_schedule | A backup schedule defines the frequency and timing of data backups. | `string` | ` ` | no |
+| backup_retention | Backup retention refers to the rules that determine how long backups are stored before being deleted. The unit is days | `numeric` | ` ` | no |
```

Comment:
Should be `number`, not `numeric`. See: https://developer.hashicorp.com/terraform/language/expressions/types#number